src/database/schema/UserSchema.ts

```diff
@@ -182,14 +182,14 @@ class UserSchema implements BaseDbSchema {
     }
   };
 
-  isUserExists = async (
+  static isUserExists = async (
     db: SQLiteDatabase,
     userId: string,
     channelId: string,
     transaction?: Transaction
   ): Promise<boolean> => {
     try {
-      const query = `SELECT * FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
+      const query = `SELECT user_id FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on optimizing the SQL query in `isUserExists` method. By selecting only the `user_id` instead of all columns (`*`), you are reducing the amount of data that needs to be fetched from the database, which can lead to performance improvements especially when dealing with large datasets.
```diff
-      const query = `SELECT * FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
+      const query = `SELECT user_id FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
```
Also, changing the `isUserExists` method to a static method allows it to be called directly on the class itself without needing to instantiate an object of the class. This is a good practice for utility methods like this one that don't rely on any instance variables.


<!-- This is an auto-generated comment by OSS CodeRabbit -->